modules/tinymce/changelog.txt

```diff
@@ -14,6 +14,7 @@ Version 5.7.0 (TBD)
     Changed the type signature for `editor.selection.getRng()` incorrectly returning `null` #TINY-6843
     Changed some `SaxParser` regular expressions to improve performance #TINY-6823
     Fixed `col` elements incorrectly transformed to `th` elements when converting columns to header columns #TINY-6715
+    Fixed a number of table operations not working when selecting 2 table cells on Mozilla Firefox #TINY-3897
```

Comment:
Do we normally refer to it as "Mozilla Firefox" in the docs?

Reply:
Style guide says to use the full product name, eg `Microsoft Internet Explorer`: https://www.tiny.cloud/docs/configure/contributing-docs/#usepropernamesforprogramsandlanguages